Sources/SubscriberAttributes/SubscriberAttributesManager.swift

```diff
@@ -232,6 +232,17 @@ class SubscriberAttributesManager {
             network: network.rawValue)
         setAttributes(convertedAttribution, appUserID: appUserID)
     }
+
+}
+
+extension SubscriberAttributesManager: AttributeSyncing {
+
+    func syncSubscriberAttributesIfNeeded(currentAppUserID: String, completion: @escaping (() -> Void)) {
```

Comment:
I kinda think it should be `syncSubscriberAttributes` since the `ifNeeded` implies that it might not sync. The current implementation does seem to always sync each time

Reply:
This is the same name as `Attribution.syncSubscriberAttributesIfNeeded`, it comes from the fact that there might not be attributes to sync for the given user, but maybe there's a better way to phrase that?
Or perhaps, that's obvious (if there's nothing to sync, welp, then do do anything).

I'll remove that suffix 👍🏻